etna/analysis/plotters.py

```diff
@@ -536,3 +536,48 @@ def plot_clusters(
             centroid = centroids_df[cluster, "target"]
             axs[h][w].plot(centroid.index.values, centroid.values, c="red", label="centroid")
         axs[h][w].legend()
+
+
+def plot_time_series_with_change_points(
+    ts: "TSDataset",
+    change_points_dict: Dict[str, List[np.datetime64]],
+    segments: Optional[List[str]] = None,
+    columns_num: int = 2,
+    figsize: Tuple[int, int] = (10, 5),
+):
+    """Plot segments with their trend change points.
+
+    Parameters
+    ----------
+    ts:
+        TSDataset with timeseries
+    change_points_dict:
+        dictionary derived from function for finding trend change points
```

Comment:
May be it is better to write that it is just dictionary with trend change points for the segments and then write that we have function  that can be used to obtain them